modules/ROOT/pages/io-connectors.adoc

```diff
@@ -106,7 +106,7 @@ configs:
 
 == Troubleshooting
```

Comment:
The comment is really a general comment for the entire "io-connectors.adoc" doc. I personally think we need to revisit this particular piece of document because the existing doc spends a lot of time only on DS enhanced Pulsar C* sink connector which is only ONE connector that Luna Streaming supports.

If we want to do a proper fix, I would recommend the following document structure
1) The general concept of Pulsar I/O connectors and built-in list of available connectors (can point to OSS Pulsar doc)
2) List what enhanced Pulsar I/O connectors are available in Luna Streaming and for each enhanced I/O connector, provide an example to configure, deploy, and execute it.
3) Compare Pulsar I/O connectors with Kafka Connect and explain how DataStax Luna streaming can help reduce the gap of existing huge Kafka connect packages (through Pulsar Kafka Connect Adaptor)

To make things simple, we can start with item 1) and for item 2) we only make a list and we can add examples in the future step by step. Item 3) can leave to the future.